osu.Game/Screens/Play/Player.cs

```diff
@@ -498,8 +498,24 @@ public override void OnEntering(IScreen last)
                 .Delay(250)
                 .FadeIn(250);
 
-            Background.EnableUserDim.Value = true;
-            Background.BlurAmount.Value = 0;
+            var screenOverride = Mods.Value.OfType<IApplicableToScreen>();
```

Comment:
Was thinking that if more than one mod in the list had the `IApplicableToScreen` modifier it threw an exception. In this case line 503 would become a switch with `0`, `1` and `default`, with default throwing the exception.

Any ideas on an appropriate exception to throw, or should I make my own, like `IncompatableModSelection`?

Reply:
there's already a system in place for incompatible mods (see `IncompatibleMods`). this kind of logic should not exist.